hammr/utils/generate_utils.py

```diff
@@ -270,6 +270,14 @@ def generate_docker(image, builder, installProfile, api=None, login=None):
     image.compress = True
     return image, installProfile
 
+def generate_openshift(image, builder, installProfile, api=None, login=None):
```

Comment:
you could add unit tests for this method, we already have some for other builders format.

Reply:
And also, our wonderful linter is asking for a renaming of installProfile to install_profile and he is right

Reply:
Done